listenbrainz/custom_config.py.sample

```diff
@@ -36,6 +36,12 @@ RABBITMQ_USERNAME = "guest"
 RABBITMQ_PASSWORD = "guest"
 RABBITMQ_VHOST = "/"
 
+# RabbitMQ exchanges and queues
+INCOMING_EXCHANGE = "incoming_exchange"
```

Comment:
I think we should ditch the _exchange and _queue bits. I believe they are separate namespaces and therefore a bit redundant. And if we change the names of the queues, I fear that we'll lose some listens when we deploy this.